src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -55,6 +55,7 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
+    <a href="/activity"> Activity Feed </a>
```

Comment:
super nit: Remove the spaces on either side of "Activity Feed" to be consistent with the rest of the links.

Same for the other jsp files